src/components/EventTabs.spec.tsx

```diff
@@ -24,47 +21,36 @@ describe('EventTabs', () => {
             .then((data : {data: ZetkinEvent[]}) => {
                 dummyBookedEvents = data.data;
             });
-    });
-
-    beforeEach(()=> {
         cy.fixture('dummyEvents.json')
             .then((data : {data: ZetkinEvent[]}) => {
-                dummyTimeRange = {
-                    later: data.data,
-                    today: data.data,
-                    tomorrow: data.data,
-                    week: data.data,
-                };
+                dummyEvents = data.data;
             });
     });
 
-    it('only shows a placeholder if there are no upcoming events', () => {
-        mountWithProviders(
-            <EventTabs
-                bookedEvents={ dummyBookedEvents }
-                eventResponses={ dummyEventResponses }
-                onSignup={ () => null  }
-                onUndoSignup={ () => null  }
-                timeRange={ dummyTimeRange }
-            />,
-        );
+    it('contains tabs for time filtering on today and whis week', () => {
+        const filterDummyEvents = [
```

Comment:
The name of this variable is confusing to me. What does `filter` mean in this context?